app/components/lazy-minted/LazyMinted.tsx

```
@@ -0,0 +1,107 @@
+                </td>
+                <td className="border border-slate-700 px-4 py-1">
+                  <video
+                    src={
+                      nft.metadata.animation_url
+                        ? parseIpfsUri(nft.metadata.animation_url)
+                        : ''
+                    }
+                    aria-label={`Preview of ${nft.metadata.name || 'NFT'}`}
+                    playsInline
+                    width={180}
+                  ></video>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)